ffi/src/lib.rs

```diff
@@ -280,13 +496,39 @@ pub unsafe extern "C" fn query_builder_execute_scalar(query_builder: *mut QueryB
     Box::into_raw(Box::new(extern_result))
 }
 
+/// Executes a query and returns the results as a Coll.
```

Comment:
Through-out: I think all of these will be richer if they refer to `QueryResults::Coll` and get hyperlinked.